### PR TITLE
fix markdown syntax and broken links

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: docs/analog/commands/DataSegment.md [hunk 1]
```diff
@@ -65,13 +65,13 @@ Denotes the index or position of a particular segment within the entire message.
...
-| ----------------------- | ---------------------------------------------------------------------- | ------------ | ------------------ |
-| command id              | `30`                                                                   |              | `0x1e`             |
-| command size            | `9`                                                                    |              | `0x09`             |
-| segmentation session id | `35`                                                                   |              | `0x23`             |
-| segment flags           | last segment: `true` <br> segments number: `1` <br> segment index: `1` | `0b10010001` | `0x91`             |
-| data                    | `23 10 10 07 00 00 00`                                                 |              | `0x23101007000000` |
+| Field                   | Value                                                                    | Bits         | Hex                |
+|-------------------------|--------------------------------------------------------------------------|--------------|--------------------|
```
Comment (DeadByElpy) [inline docs/analog/commands/DataSegment.md:69]: all tables used spaces between columns, like this

```
| Field                   | Value                                                                  | Bits         | Hex                |
| ----------------------- | ---------------------------------------------------------------------- | ------------ | ------------------ |
| command id              | `30`                                                                   |              | `0x1e`             |
```

use autoformatting from https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one extension

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: docs/analog/commands/uplink/DayMC.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ This command is generated by the sensor and sent within a specified period witho
...
-| `1..5`     | [extended value](../../types.md#extended-value)              | [channels bit set](../../types.md#channels-bit-set) |
-| `[1..5]*n` | sequence of [extended values](../../types.md#extended-value) | [channel values sequence](#channel-values)          |
+| Size       | Type                                                         | Field                                                         |
+| ---------- | ------------------------------------------------------------ | ------------------------------------------------------------- |
+| `1`        | `uint8`                                                      | command id = `0x16`                                           |
+| `1`        | `uint8`                                                      | command size (dynamic, `4+`)                                  |
+| `2`        | [packed date](../../types.md#packed-date)                    | [date](#date)                                                 |
+| `1..5`     | [extended value](../../types.md#extended-value)              | [channels bit set](../../parameter-types.md#channels-bit-set) |
```
Comment (DarkPark) [inline docs/analog/commands/uplink/DayMC.md:17]: why change to parameter-types.md?

and in all other places

Comment (aleks-and) [reply docs/analog/commands/uplink/DayMC.md]: I made a mistake, I put it back the way it was

File: docs/mtx/result-codes.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-| `0x95`  | `BAD_SALDO_WRITE`                        |                                                                                                                                           |
-| `0x97`  | `BLOCKED_METER`                          |                                                                                                                                           |
-| `0x98`  | `UNENCRYPTED_COMMAND_DISABLED`           |                                                                                                                                           |
-| `0x99`  | `TIME_CORRECTION_FAILURE`                | correction more than once per period (`opParams.timeCorrectPeriod`) is forbidden                                                          |
-| `0x9a`  | `INVALID_CORRECTION_INTERVAL`            | correction interval (`opParams.deltaCorMin`) exceeded                                                                                     |
-| `0x9b`  | `TIME_CORRECTION_OUT_HALF_HOUR_DISABLED` | time correction with crossing the boundary of the current half-hour is prohibited<br> in case `opParams.timeCorrectPassHalfhour` is false |
```
Comment (DarkPark) [inline docs/mtx/result-codes.md:25]: need to fix formatting

File: docs/mtx/commands/SetDisplayParam.md [hunk 4]
```diff
@@ -18,12 +18,12 @@ The command access level is [READ_WRITE](../basics.md#command-access-level).
...
-| command size | `5`                                                                                   | `0x05`       |
-| display mode | additional (`1`)                                                                      | `0x01`       |
-| displays     | `TOTAL_ACTIVE_ENERGY`<br>`CURRENT_IN_PHASE`<br>`SOFTWARE_VERSION`<br>`OPTOPORT_SPEED` | `0x030a021f` |
+| Field        | Value                                                                                    | Hex          |
+| ------------ | ---------------------------------------------------------------------------------------- | ------------ |
+| command id   | `93`                                                                                     | `0x5d`       |
+| command size | `5`                                                                                      | `0x05`       |
+| display mode | additional (`1`)                                                                         | `0x01`       |
+| displays     | `TOTAL_ACTIVE_ENERGY` <br/> `CURRENT_IN_PHASE` <br/> `SOFTWARE_VERSION` <br/> `OPTOPORT_SPEED` | `0x030a021f` |
```
Comment (DarkPark) [inline docs/mtx/commands/SetDisplayParam.md:26]: and here

File: docs/mtx/result-codes.md (hunk 3 above)
Comment (aleks-and) [reply docs/mtx/result-codes.md:25]: fixed

File: docs/mtx/commands/SetDisplayParam.md (hunk 4 above)
Comment (aleks-and) [reply docs/mtx/commands/SetDisplayParam.md:26]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**